### PR TITLE
Nada additions and fixes, docs and logging in general for Nillion

### PR DESCRIPTION
**Pull request by amardeepio**
[@vishakh's edits]

Made some changes on top of @amardeepio's changes. The net effect is:

:computer: We have an additional Nada program for checking the muscle performance trait
:bug: The thrombosis Nada program now looks for the correct SNP and genotype
:writing_hand: More comments and logging have been added across the board
:x:  The new muscle performance trait has not been included in the API as we do not want to change the DNA wallet format at this late juncture. 

I have asked a question on Discord about [multiple programs using the same secrets](https://discord.com/channels/554623348622098432/1224124237929779310/1230679099308245095). If the Nillion team gives us a simple solution, we can revisit adding muscle performance to the API. 

I looked pretty extensively through the Nillion documents and did not find any real clues to solving the problem.

### CODE REVIEW

File: services/nillion-interactor/programs/muscle-perform.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+def nada_main():
+
+    party1 = Party(name="Party1")
+
+
+    snp = SecretInteger(Input(name="snp", party=party1))
+    genotype = SecretInteger(Input(name="genotype", party=party1))
+
+    target_snp = Integer(6)
+    target_genotype = Integer(12)
```
Comment (vishakh) [inline services/nillion-interactor/programs/muscle-perform.py:12]: You shouldn't copy the logic exactly from `thrombosis.py`. Here we are testing for a different condition, i.e. muscle performance which looks at the SNP `rs1815739` and the genotype `TT` to identify endurance athletes.

File: services/nillion-interactor/programs/muscle-perform.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    eq2 = (genotype >= target_genotype).if_else((target_genotype >= genotype).if_else(Integer(1), Integer(0)), Integer(0))
+
+
+    total = eq1 + eq2
+
+    is_successful = (total >= Integer(2)).if_else((Integer(2) >= total).if_else(Integer(1), Integer(0)), Integer(0))
+
+    result = is_successful.reveal()
+
+    return [Output(result, "Result", party1)]
```
Comment (vishakh) [inline services/nillion-interactor/programs/muscle-perform.py:24]: Leave a comment here about what the return value means.

**Review by vishakh — CHANGES_REQUESTED** [review]

File: services/nillion-interactor/.env
[the remark this answers is not in the document]
Comment (amardeepio) [reply services/nillion-interactor/.env:1]: reverted changed

File: services/nillion-interactor/app.py [hunk 3]
```diff
@@ -157,6 +157,13 @@ async def thrombosis():
     result = await asyncio.wait_for(compute_on_nillion(store_id), timeout=120)
     return jsonify(result)
 
+@app.route('/computations/muscle-perform', methods=['POST'])
```
[the remark this answers is not in the document]
Comment (amardeepio) [reply services/nillion-interactor/app.py:160]: done.

File: services/nillion-interactor/app.py [hunk 4]
```diff
@@ -157,6 +157,13 @@ async def thrombosis():
...
+async def muscle_perform():
```
[the remark this answers is not in the document]
Comment (amardeepio) [reply services/nillion-interactor/app.py:161]: Added the requests for muscle performance using `muscle_performance_store_id`

File: services/nillion-interactor/programs/muscle-perform.py (hunk 1 above)
Comment (amardeepio) [reply services/nillion-interactor/programs/muscle-perform.py:12]: Changed thrombosis mapping to 12th SNP and 7th genotype ie `rs6025` and `GG`.
for muscle performance changed mapping to 11th SNP and 9th genotype ie `rs1815739` and `TT`.

File: services/nillion-interactor/programs/muscle-perform.py (hunk 2 above)
Comment (amardeepio) [reply services/nillion-interactor/programs/muscle-perform.py:24]: Here return value means either 1 or 0 based on matching it performed when we stored secret on nillion using `secretinteger` in` store nillion ` in `app.py `file.

**Pull request merged**